src/components/CountrySelector/style.scss

```
@@ -124,6 +124,8 @@ $_sdk-option-border-bottom: 0;
 .custom__input {
   background-color: transparent;
   position: relative;
+  /* font size for form elements (input, select, textarea) must be at least 16px otherwise iOS Safari intervenes & forces UI to zoom in */
+  font-size: 16 * $unit;
```

We already have a `$font-size-base` defined [here](https://github.com/onfido/onfido-sdk-ui/blob/dac730f897d15fab17b0e09be2b64eb91bf3a746/src/components/Theme/constants.scss#L13), is it possible to reuse it?
```suggestion
  font-size: $font-size-base;
```

---

Good catch. I was looking for this but somehow didn't find it.